src/containers/QueryEditor/YDBQueryEditor.tsx

```diff
@@ -67,40 +68,36 @@ export function YDBQueryEditor({ query: baseQuery, onChange, onRunQuery, datasou
 
   return (
     <DatasourceProvider datasource={datasource}>
-      <EditorHeightProvider>
-        <BuilderSettingsProvider builderOptions={builderOptions}>
-          <TablesProvider>
-            <Form onSubmit={onRunQuery} maxWidth="none">
-              {() => (
-                <React.Fragment>
-                  <div className={styles.Common.inlineFieldWithAddition}>
-                    <QueryTypeSwitcher
-                      queryType={queryType}
-                      onChange={handleChangeQueryType}
-                      shouldConfirm={rawSql !== rawSqlBuilder}
-                    />
-                    {queryType === 'sql' && <SqlEditorHeightInput />}
-                    {queryType === 'builder' && <QueryBuilderSettings />}
-                  </div>
-                  <QueryFormatSelect format={queryFormat} onChange={handleChangeQueryFormat} />
-                  {queryType === 'builder' ? (
-                    <QueryBuilder
-                      query={query}
-                      onChange={handleChangeQueryAttribute<YDBBuilderQuery>}
-                    />
-                  ) : (
-                    <SqlEditor
-                      onChange={handleChangeQueryAttribute<YDBSQLQuery>}
-                      query={query}
-                    />
-                  )}
-                  <Button type="submit">Run Query</Button>
-                </React.Fragment>
-              )}
-            </Form>
-          </TablesProvider>
-        </BuilderSettingsProvider>
-      </EditorHeightProvider>
+      <VariablesProvider>
```

Comment:
only SqlEditor use it, as far i can see
no need wrap all form in provider

Reply:
`useVariables` is also used in `QueryBuilder` in `Filters`.